crates/astria-core/src/execution/v1alpha2/mod.rs

```diff
@@ -104,7 +113,7 @@ impl Protobuf for GenesisInfo {
                  under the hood",
             );
         Self::Raw {
-            rollup_id: Bytes::copy_from_slice(rollup_id.as_ref()),
+            rollup_id: Some(rollup_id.to_raw()),
```

Comment:
Note that there is also `into_raw`. They do the same thing here, so it doesn't matter. But sometimes taking owernship allows some optimizations (like avoiding another allocation... if one remembers to implement it that way).

Reply:
Got it! So for future reference if something is not a shared object then `into_raw()` may be a better choice?

Reply:
I'll note that this is within a `to_raw` function. I have generally tried to keep the `to_raw` calling other `to_raw` and vice versa.